resources/chsec.rb

```diff
@@ -15,14 +15,48 @@
 #
 
 property :file_name, String, name_property: true, identity: true
-property :attributes, Hash
-property :stanza, String, desired_state: false
+property :attributes, Hash, required: true
+property :stanza, String, desired_state: false, required: true
```

Comment:
converting properties to required would be a major change as it would break folks using this resource previously who didn't have it included in their wrapper